apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -304,6 +334,23 @@ const Ticket = new GQLListSchema('Ticket', {
                         }
                     }
                 }
+                const propertyChanged = resolvedData['property'] && resolvedData['property'] !== get(existingItem, 'property')
+                const propertyAddressChanged = resolvedData['propertyAddress'] && resolvedData['propertyAddress'] !== get(existingItem, 'propertyAddress')
```

Comment:
I don't think that we need to give anyone access to change the `propertyAddress` directly. 
It's auto-generated field.

Reply:
Right now you cannot change it it manually to anything except to current ticket property.
Reason, why I left this editable: 
Somehow in db will be ticket with property with `address` `A`, but `propertyAddress` is `B`.
Then it can be fixed manually by setting `propertyAddress` to `A` (and nothing else is not allowed and covered by tests)

Reply:
But you can use `afterChange` hook like in Property model. Or you can use `resolveInput` hook like in createdAt, updatedAt, createdBy, updatedBy, v and so on fields work.

Reply:
Probably, it's better to use `resolveInput` because it allows you to avoid the redundant updates.

Reply:
didn't understand what do you want me to change?

Current logic of validation is: if at least 1 of 3 fields was changed (property link, address or meta), make sure it matches....otherwise throw error

`afterChange` is not an option, since we cannot prevent update at this stage, and `resolveInput` is not an option, since we cannot find property by other 2 fields

Reply:
And logic of resolvers are:
If detected changes in property-link, set meta and address from new property

If address of current property is changed, make sure to update fields in linked tickets

Reply:
Moved logic to model resolveInput....set up fields accesses to remove this fields from inputs